gamemodes/terrortown/gamemode/server/sv_weaponry.lua

```diff
@@ -410,9 +410,11 @@ end
 -- @note Note that this is called both when a @{Player} spawns and when a round starts
 -- @hook
 -- @realm server
--- @ref https://wiki.garrysmod.com/page/GM/PlayerLoadout
+-- @ref https://wiki.facepunch.com/gmod/GM:PlayerLoadout
 -- @local
-function GM:PlayerLoadout(ply)
+function GM:PlayerLoadout(ply, isRespawn)
+	print("loadout: " .. tostring(isRespawn))
```

Comment:
This print doesn't say much for the person that receives it

Reply:
this is a debug print by me, will be removed